artemis-server/src/main/java/org/apache/activemq/artemis/core/management/impl/ActiveMQServerControlImpl.java

```diff
@@ -2651,7 +2651,7 @@ public void buildSessionJSON(JsonArrayBuilder array, ServerSession sess) {
       JsonObjectBuilder obj = JsonLoader.createObjectBuilder().add("sessionID", sess.getName()).add("creationTime", sess.getCreationTime()).add("consumerCount", sess.getServerConsumers().size());
 
       if (sess.getValidatedUser() != null) {
-         obj.add("principal", sess.getValidatedUser());
+         obj.add("validatedUser", sess.getValidatedUser());
```

Comment:
I'm fine with either validated user or principal. I think it was Justin (?) who said to keep "validated user" to keep things consistent.